app/src/common/shared/com/igalia/wolvic/telemetry/OpenTelemetry.java

```diff
@@ -0,0 +1,91 @@
+package com.igalia.wolvic.telemetry;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import android.app.Application;
+import android.os.Bundle;
+
+import com.igalia.wolvic.BuildConfig;
+import com.igalia.wolvic.VRBrowserApplication;
+
+import java.util.concurrent.Executor;
+
+import io.opentelemetry.android.OpenTelemetryRum;
+import io.opentelemetry.android.OpenTelemetryRumBuilder;
+import io.opentelemetry.android.config.OtelRumConfig;
+import io.opentelemetry.android.features.diskbuffering.DiskBufferingConfiguration;
+import io.opentelemetry.android.instrumentation.activity.ActivityLifecycleInstrumentation;
+import io.opentelemetry.android.instrumentation.sessions.SessionInstrumentation;
+import io.opentelemetry.api.trace.SpanBuilder;
+import io.opentelemetry.exporter.logging.LoggingSpanExporter;
+
+public class OpenTelemetry implements ITelemetry {
+    private final Application application;
```

Comment:
We use to define private attributes with the "m" prefix; any reason why we are not following the usual coding-style here ?